R/module_teal_data.R

```diff
@@ -18,15 +18,17 @@
 #' `teal` (observers in `srv_teal`) always waits to render an app until `reactive` `teal_data` is
 #' returned. If error 2-4 occurs, relevant error message is displayed to app user and after issue is
 #' resolved app will continue to run. `teal` guarantees that errors in a data don't crash an app
-#' (except error 1). This is possible thanks to `.fallback_on_failure` which returns input-data
-#' when output-data fails
+#' (except error 1). This is possible by catching the error and returning an empty `teal_data` object.
+#' The error message is displayed in the app and the module will eventually get an empty `teal_data`.
```

Comment:
This part of the documentation is the remaining of the solution implemented by Vedha in [this PR](https://github.com/insightsengineering/teal/pull/1330/) but is no longer true and should be reverted.

Reply:
Fixed with https://github.com/insightsengineering/teal/pull/1341/commits/4cf27304ddd8f3ddf0746bd2c829ff822fff8c3a